src/Modal/Modal.js

```diff
@@ -73,8 +75,9 @@ const ModalContent = createComponent({
   `,
 });
 
-function Modal({ children, title, animationDuration, showClose, onClose, open, ...props }) {
+function Modal({ children, title, animationDuration, showClose, onClose, open, blur, blurTarget, ...props }) {
   const [isOpen, setOpen] = useState(open);
+  const [blurNode, setBlurNode] = useState();
```

Comment:
Have you considered using a ref to store the `blurTarget` since it will be a DOM node? I don't think we need to re-render the component using `setBlurNode` because all we want to do is keep track of this node across renders, which `useRef` is designed to do. LMK your thoughts.

Reply:
What do you mean exactly? I've only used refs for nodes created by the component, is there a way to use `createRef` to target a node outside of the component?

Reply:
```jsx
function DumbModal() {
  const blurNode = useRef();

  const toggleBlur = () => {
    if (blur && blurNode.current) {
      blurNode.current.style.filter = open ? 'blur(1px)' : null;
    }
  };

  useEffect(() => {
    const target = document.querySelector(blurTarget);

    if (target) {
      blurNode.current = target;
    }
  }, []);

}
```

Reply:
Hmm. Makes sense. Refs should be used if needing to touch the DOM too. Great suggestion!